perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_resource_attribute_def_virt_unixGID.java

```diff
@@ -62,6 +60,7 @@ public Attribute fillAttribute(PerunSessionImpl sess, Group group, Resource reso
 		}
 
 		try {
+			sess.getPerunBl().getAttributesManagerBl().checkAttributeSyntax(sess, group, gidAttribute);
```

Comment:
Same here. You added checkAttributeSyntax but in the fillAttribute method in virt_googleGroupNames there is just checkAttributeSemantics.

Reply:
Also I would use forceCheckAttributeSyntax here. Same for virt_googleGroupName, and you can change them to "force" version in both check and fill methods.